backend-server/src/api/routes/event.ts

```diff
@@ -0,0 +1,41 @@
+import { Timestamp } from "@firebase/firestore";
+import express from "express";
+import { report } from "process";
+import {
+  addEvent, bookEvent, getEvent,
+} from "../controllers/eventController";
+
+export const eventRouter = express.Router();
+
+eventRouter.post("/", async (req, res) => {
+  const { organization, lead1, lead2, intervieweeEmail, length, expires} = req.body;
+  try {
+    const ret = await addEvent(organization, lead1, lead2, intervieweeEmail, length, expires);
+    res.send(ret);
+  } catch (err) {
+    res.send(`error processing request: ${err}`);
+  }
+});
+
+eventRouter.get('/', async (req, res) => {
+  // const { organization, eventUID } = req.body;
+  let organization:any= req.query.organization;
```

Comment:
we should try to avoid using any whenever possible, because that defeats the purpose of having types. (this should be fixed by the previous comments about having a data model for the request body though)